src/app/(proper_react)/(redesign)/(authenticated)/user/(dashboard)/dashboard/View.tsx

```diff
@@ -171,12 +176,20 @@ export const View = (props: Props) => {
     return timestampB - timestampA;
   });
 
+  const dataBrokersResultsRemovalUnderMaintenance =
+    props.userScanData.dataBrokersRemovalUnderMaintenance || [];
```

Comment:
It is safer to use the `??` to check if `props.userScanData.dataBrokersRemovalUnderMaintenance` is `null` or `undefined`.

```suggestion
    props.userScanData.dataBrokersRemovalUnderMaintenance ?? [];
```